Scientific_workflows/DEAWaterbodies/DEAWaterbodiesToolkit/WOfSImputation.ipynb

```diff
@@ -0,0 +1,1810 @@
+{
```

Comment:
<p>Can you clarify this? Maybe 'Where there are not many observations, filling with the mean water level will cause the waterbody to appear as if it spends most of the time at the mean water level' - Is this valid? </p><p> </p><p>...Did you look at filling with a rolling mean on, say, a seasonal scale? I guess that's just retrending on a different scale...</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GeoscienceAustralia/dea-notebooks/pull/713/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='70'/>

Reply:
<p>Good description, I'll add that. I didn't try a rolling mean - I can definitely do that and add it in.</p>

Reply:
<p>OK, this turns out to be terribly slow, so I'm going to either not do it or defer it to another time.</p>

Reply:
No worries! Leave it out :)